package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "fh-mbaas-client",
   "description": "FeedHenry MBaaS Client",
-  "version": "0.15.0",
+  "version": "0.15.1",
   "author": "FeedHenry",
   "main": "index.js",
   "directories": {
     "lib": "lib"
   },
   "engines": {
-    "node": ">=0.8 < 0.11"
+    "node": "4.4"
```

Comment:
@nialldonnellyfh It does, but [it looks like it just does npm install](https://github.com/feedhenry/fh-mbaas-client/blob/master/.travis.yml) and not run the tests. Maybe that's enough though? Also, travis is sssslllloooooowww (yeah, I know bob has its own set of problems, but they're within our control to fix :))